packages/backend/src/server/api/endpoints/admin/show-user.ts

```diff
@@ -23,9 +23,12 @@ export const paramDef = {
 
 // eslint-disable-next-line import/no-default-export
 export default define(meta, paramDef, async (ps, me) => {
-	const user = await Users.findOneBy({ id: ps.userId });
+	const [user, profile] = await Promise.all([
+	 Users.findOneBy({ id: ps.userId }),
+	 UserProfiles.findOneBy({ userId: ps.userId })
```

Comment:
sorry mixed up tabs and spaces, my suggested change breaks the lint

```suggestion
		Users.findOneBy({ id: ps.userId }),
		UserProfiles.findOneBy({ userId: ps.userId })
```